src/view.c

```diff
@@ -408,7 +383,6 @@ _minimize(struct view *view, bool minimized)
 	view->minimized = minimized;
 	if (minimized) {
 		view->impl->unmap(view, /* client_request */ false);
-		view_defocus(view);
```

Comment:
I checked and both of the `unmap()` implementations already call `desktop_focus_topmost_mapped_view()`, so the `view_defocus()` call was redundant and is safe to remove.